pulsar-io/debezium/core/src/main/java/org/apache/pulsar/io/debezium/PulsarDatabaseHistory.java

```diff
@@ -61,26 +66,34 @@
         .withValidation(Field::isRequired);
 
     public static final Field SERVICE_URL = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "pulsar.service.url")
-        .withDisplayName("Pulsar broker addresses")
+        .withDisplayName("Pulsar service url")
         .withType(Type.STRING)
         .withWidth(Width.LONG)
         .withImportance(Importance.HIGH)
         .withDescription("Pulsar service url")
         .withValidation(Field::isRequired);
```

Comment:
we can set either this or the following field. so they might not be `required` now.